qa/tasks/thrashosds-health.yaml

```diff
@@ -16,3 +16,10 @@ overrides:
       - \(REQUEST_SLOW\)
       - \(TOO_FEW_PGS\)
       - slow request
+      - \(MON_DOWN\)
+      - osds down
+      - mons down
+      - flag\(s\) set
+      - out of quorum
+      - PG_
```

Comment:
Is that one restrictive enough?

Reply:
@ronen-fr I can make it more specific

Reply:
@ronen-fr I think Sam's comment makes sense. Let's keep it as is, especially since we whitelist that exact string in many of our other tests:
```
$ git grep "PG_"
basic/tasks/rados_api_tests.yaml:    - \(PG_AVAILABILITY\)
basic/tasks/rados_api_tests.yaml:    - \(PG_DEGRADED\)
basic/tasks/rados_cls_all.yaml:    - \(PG_AVAILABILITY\)
basic/tasks/rados_python.yaml:    - \(PG_
basic/tasks/repair_test.yaml:      - \(PG_
basic/tasks/scrub_test.yaml:    - \(PG_
dashboard/tasks/dashboard.yaml:        - \(PG_
mgr/tasks/crash.yaml:        - \(PG_
mgr/tasks/failover.yaml:        - \(PG_
mgr/tasks/insights.yaml:        - \(PG_
mgr/tasks/module_selftest.yaml:        - \(PG_
mgr/tasks/progress.yaml:        - \(PG_
mgr/tasks/prometheus.yaml:        - \(PG_
mgr/tasks/workunits.yaml:        - \(PG_
monthrash/ceph.yaml:# slow mons -> slow peering -> PG_AVAILABILITY
monthrash/ceph.yaml:      - \(PG_AVAILABILITY\)
monthrash/workloads/rados_api_tests.yaml:      - \(PG_
monthrash/workloads/rados_mon_workunits.yaml:    - \(PG_
multimon/tasks/mon_clock_with_skews.yaml:    - \(PG_
multimon/tasks/mon_recovery.yaml:      - \(PG_AVAILABILITY\)
objectstore/backends/ceph_objectstore_tool.yaml:      - \(PG_
perf/ceph.yaml:      - \(PG_
rest/mgr-restful.yaml:      - \(PG_
singleton-bluestore/all/cephtool.yaml:    - \(PG_
singleton-bluestore/all/cephtool.yaml:    - \(SMALLER_PG_NUM\)
singleton-nomsgr/all/balancer.yaml:      - \(PG_AVAILABILITY\)
```